vocabularies/Common.xml

```diff
@@ -553,34 +553,6 @@ The minimum structure is
         </Annotation>
       </Term>
 
-      <Term Name="WeakReferentialConstraint" Type="Common.WeakReferentialConstraintType" Nullable="false" AppliesTo="EntitySet">
```

Comment:
Why is this term deleted in this PR?

Reply:
It was an early attempt at solving the soft-link problem, which is now replaced with the SONT-reference